packaging/suse/Dockerfile

```diff
@@ -5,7 +5,8 @@
 #!UseOBSRepositories
 
 FROM bci/rust:1.66 AS release
-ADD wanda.tar.gz /build/
+ADD wanda.tar.gz premium-checks.tar.gz* /build/
```

Comment:
It wouldn't be better:
```
ADD wanda.tar.gz /build/
ADD premium-checks.tar.gz* /build/wanda/
```
As the `ADD` command extract the tar file in the destination folder.
Didn't test it though

Reply:
Tested and doesn't work when `premium-checks.tar.gz` is not there. Docker wants at least the first item for an `ADD` to actually be there.

Reply:
ok!
Thank you for trying out